product_docs/docs/biganimal/release/overview/03_security/01_securing_your_cluster.mdx

```diff
@@ -0,0 +1,175 @@
+
+1. Launch the Azure Policy service in the Azure portal by searching for and selecting the *Policy* service in the search box at the top.
+
+1. Select **Assignments** on the left side of the Azure Policy page. 
+
+1. Select **Assign Policy** from the top of the Policy - Assignments page.
+
+1. On the Assign Policy page, set the Scope by selecting the ellipsis and then selecting the applicable subscription. A scope determines what resources or grouping of resources the policy assignment gets enforced on. Then use the **Select** button at the bottom of the Scope page.
+
+1. Select the Policy definition ellipsis to open the list of available definitions. Azure Policy comes with built-in policy definitions you can use. Many are available, such as:
+ - Enforce tag and its value
+ - Apply tag and its value
+ - Inherit a tag from the resource group if missing
+ - For a partial list of available built-in policies, see Azure Policy samples.
+
+1. Search through the policy definitions list to find the ASC policy policy definition you need. Select that policy and then use the Select button.
+
+1. The Assignment name is automatically populated with the policy name you selected, but you can change it. For this example. You can also add an optional Description. The description provides details about this policy assignment. Assigned by will automatically fill based on who is logged in. This field is optional, so custom values can be entered.
+
+1. Leave policy enforcement Enabled. For more information, see Policy assignment - enforcement mode.
+
+1. Select Next at the bottom of the page or the Parameters tab at the top of the page to move to the next segment of the assignment wizard.
+
+1. If the policy definition selected on the Basics tab included parameters, they are configured on this tab. Since the Audit VMs that do not use managed disks has no parameters, select Next at the bottom of the page or the Remediation tab at the top of the page to move to the next segment of the assignment wizard.
+
+1. Leave Create a Managed Identity unchecked. This box must be checked when the policy or initiative includes a policy with either the deployIfNotExists or modify effect. As the policy used for this quickstart doesn't, leave it blank. For more information, see managed identities and how remediation security works.
+
+1. Select Next at the bottom of the page or the Non-compliance messages tab at the top of the page to move to the next segment of the assignment wizard.
+
+1. Set the Non-compliance message to Virtual machines should use a managed disk. This custom message is displayed when a resource is denied or for non-compliant resources during regular evaluation.
+
+1. Select Next at the bottom of the page or the Review + Create tab at the top of the page to move to the next segment of the assignment wizard.
+
+1. Review the selected options, then select Create at the bottom of the page.
+
+## Remediate non-compliant resources
```

Comment:
@drothery-edb i have reached a point where documenting steps for this sections is very difficult unless i have access to a dev build.